src/plugins/maxlength/plugin.yml

```diff
@@ -0,0 +1,13 @@
+about: |
+  Use dmenu for input of maximum length, without 
+  any menu items. Acts as a replacement for `i3-input -l`
+  Pass --maxlength=<MAXLENGTH> to use
+entry: main.rs # entry has all overriding and pub items
+
+args:
+  - maxlength:
+      help:  Auto select if there is only one match with the current input
```

Comment:
It looks like the `help` field was copied over from the autoselect plugin and was forgotten about. This should be a shorter version of the `about` field.